### DISCUSSION TITLE
why is .env file not included in the gitignore template that is automatically setup with vite setup?

### DISCUSSION THREAD

**Discussion by shradiphylleia** [vitejs/vite-plugin-react#379] [Q&A]
was wondering why the template for the gitignore not mention .env 
almost pushed it to my repo today without checking it out.

if this could be improved, please guide me to the correct repo to maybe contribute this. thanks

**Comment by ArnaudBarre** [comment]
In Vite you should use `.env.local` for things you dont want to push to the repo: https://vite.dev/guide/env-and-mode#env-variables
